dropwizard-jsonapi/src/main/java/io/rtr/jsonapi/filter/JsonApiResponseFilter.java

```diff
@@ -213,4 +216,11 @@ private boolean isApplicable(ContainerRequestContext requestContext) {
 				.stream()
 				.anyMatch(m -> JSONAPI_MEDIATYPE.equals(m));
 	}
+
+	@VisibleForTesting
```

Comment:
I'm not a fan of this anti-pattern. The test should only test the class contract defined by the public methods. Otherwise the tests are fragile.